app/models/concerns/type_of_application.rb

```diff
@@ -13,10 +13,21 @@ def can_receive_pse?
   end
 
   def initial?
-    ApplicationType.new(application_type) == ApplicationType::INITIAL
+    application_type?(:initial)
   end
 
   def returned?
     returned_at.present?
   end
+
+  def post_submission_evidence?
+    application_type?(:post_submission_evidence)
+  end
+  alias pse? post_submission_evidence?
+
+  private
+
+  def application_type?(other_type)
+    ApplicationType.new(application_type) == ApplicationType.new(other_type)
```

Comment:
Thinking about the API here this "is the application type the same" not sure I get that its a comparison method from the name. maybe `application_type_equals?` or just override the `.eql?` method on this class and do the same as we do in ValueObject?

Reply:
I've renamed it to be `application_type_eql?`